test/blackbox-tests/test-cases/pkg/depopts/gh11698.t

```diff
@@ -0,0 +1,31 @@
+Reproduce the bug in #11698
+
+  $ . ../helpers.sh
+
+  $ mkrepo
+  $ add_mock_repo_if_needed
+
+  $ mkpkg fmt
+  $ mkpkg semver
+
+  $ mkdir _dep
+  $ cat >_dep/dune-project <<EOF
+  > (lang dune 3.18)
+  > (package
+  >  (name dep)
+  >  (depopts fmt semver))
+  > EOF
+
+  $ solve_project <<EOF
+  > (lang dune 3.18)
+  > (pin
+  >  (url "$PWD/_dep")
```

Comment:
```suggestion
  >  (url "file://$PWD/_dep")
```
I know this is not required but I find it helpful to explicitly state the protocol to make the test easier to read quickly.